docs/developers/evm-tracing/live-tracing.md

```diff
@@ -292,7 +292,7 @@ Note-worthy explanations:
 First compile the source by running `make geth`. Then run the following command:
 
 ```bash
-./build/bin/geth --vmtrace --vmtrace.config '{"config": "supply-logs"}' [OTHER_GETH_FLAGS]
```

Comment:
Ah wait, I think we also renamed the config flag. Was this working for you?

Reply:
I didn't use a config, so I removed the config flag when trying my tracer

Reply:
Yep it was renamed to `--vmtrace.jsonconfig`. Can you please fix that too?
https://github.com/ethereum/go-ethereum/blob/86a1f0c39494c8f5caddf6bd9fbddd4bdfa944fd/cmd/utils/flags.go#L548

Reply:
done